app/src/main/java/net/gsantner/markor/activity/MainActivity.java

```diff
@@ -83,6 +85,7 @@
     private AppSettings _appSettings;
     private ActivityUtils _contextUtils;
     private ShareUtil _shareUtil;
+    private File _startFolder;
```

Comment:
screen rotation / background safe? (note: savedInstanced)

Reply:
Yes. `File` is serializable

Reply:
Yes but it's not serialized (put) currently right? required to save this for later on?